_pages/ndtmetrics.md

```diff
@@ -159,3 +159,146 @@ WHERE
        AND web100_log_entry.snap.State <= 11))
   LIMIT 100
 ```
+
+## Packet Retransmission Rate
+
+NDT keeps track of the number of packets retransmitted during a test, in the web100 variable `web100_log_entry.snap.SegsRetrans`.
+
+Packet retransmission is computed as the ratio between the re-transmitted packets and all the transmitted packets.
+`web100_log_entry.snap.SegsRetrans/web100_log_entry.snap.DataSegsOut`
+
+Given that the NDT server updates the web100 variables `web100_log_entry.snap.SegsRetrans` and `web100_log_entry.snap.DataSegsOut` only when sending data, **packet retransmission is only estimated for server-to-client or download tests**.
+
+It is also possible to also measure the byte retransmission, as `web100_log_entry.snap.OctetsRetrans/web100_log_entry.snap.DataOctetsOut`
+
+The complete BigQuery query is:
+
+```sql
+#standardSQL
+SELECT
+ web100_log_entry.connection_spec.remote_ip AS remote_ip,
+ web100_log_entry.connection_spec.local_ip AS local_ip,
+ (web100_log_entry.snap.SegsRetrans / web100_log_entry.snap.DataSegsOut) AS packet_retransmission_rate
+FROM
+  `measurement-lab.release.ndt_all`
+WHERE
```

Comment:
My understanding is that the ndt_all table already filters for this 'WHERE' clause and that the base table would be the table to use this filter on. But It'd be good to confirm with @gfr10598 @stephen-soltesz and others.

Reply:
It would be great to confirm, because I had the opposite understanding about `ndt_all`. My understanding is that this is the case for `ndt_downloads` and `ndt_uploads` views.

Reply:
I need to double check, but pretty sue the filter is applied to all as well.  IIRC There is another internal view called exhaustive which does not have the filter.

Reply:
Confirmed
https://bigquery.cloud.google.com/table/measurement-lab:internal_v3_1.ndt_all?pli=1&tab=details

Reply:
Thanks @gfr10598 and @georgiamoon for reviewing. I re-read this query this morning, and since packet retransmission rate is only calculable for download tests, I'll just change the table name to `ndt_downloads` and remove the pieces of the WHERE statement that filter for valid tests, since that's already done in the views.